torchmdnet/module.py

```diff
@@ -215,11 +221,6 @@ def on_validation_epoch_end(self):
             result_dict.update(self._get_mean_loss_dict_for_type("total"))
             result_dict.update(self._get_mean_loss_dict_for_type("y"))
```

Comment:
Does this not result in an error if you do not put 0 in the losses?

Reply:
Not really, considering for example `y_weight=0.0`, this is what I get from `_get_mean_loss_dict_for_type` 

```
ANALYSIS OF TYPE total
stage train dict_keys(['mse_loss'])
stage val dict_keys(['l1_loss', 'mse_loss'])
stage test dict_keys([])
final:{'train_total_mse_loss': tensor(667.6912, device='cuda:0'), 'val_total_l1_loss': tensor(19.1803, device='cuda:0'), 'val_total_mse_loss': tensor(625.8098, device='cuda:0')}

ANALYSIS OF TYPE y
stage train dict_keys([])
stage val dict_keys([])
stage test dict_keys([])
final mean_losses:{}

ANALYSIS OF TYPE neg_dy
stage train dict_keys(['mse_loss'])
stage val dict_keys(['l1_loss', 'mse_loss'])
stage test dict_keys([])
final mean_losses:{'train_neg_dy_mse_loss': tensor(667.6912, device='cuda:0'), 'val_neg_dy_l1_loss': tensor(19.1803, device='cuda:0'), 'val_neg_dy_mse_loss': tensor(625.8098, device='cuda:0')}
```

All the dict in `self.losses `are default dict, so if a key is not present it return a empty list.